README.md

```diff
@@ -73,20 +73,29 @@ Install [Scanitizer](https://github.com/apps/scanitizer) (currently in Beta) on
 ```yaml
 - uses: azure/container-scan@v0
   with:
-    image-name: my-image
+    image-name: my-image:${{ github.sha }}
 ```
 
 ### Container scan of an image available on a private registry
 
 ```yaml
 - uses: azure/container-scan@v0
   with:
-    image-name: loginServerUrl/my-image # loginServerlUrl would be empty if it's hosted on dockerhub
+    image-name: loginServerUrl/my-image:${{ github.sha }} # loginServerlUrl would be empty if it's hosted on dockerhub

```

Comment:
I think the intention here is to define what the action interface is. `${{ github.sha }}` is an example tag, which is shown in the E2E example below.

Change this (and the other two) from `my-image:${{ github.sha }}` to `my-image:my-tag`?

Reply:
Agreed. This is not required. 

If you still feel like adding this, support this boilerplate with a build example too. Add a `run: docker build . --tag my-image:${{github.sha}}`

Reply:
The intention here was to show best practice is to tag your image with something trackable, and if, for example, you chose to use a provided GitHub Actions variable, you will need to wrap it in double braces with a dollar sign or the string will not be interpolated properly. However, I agree you may opt to show a simple example of `my-image:my-tag` on L76 so I have updated the basic example accordingly. Further, I added additional remarks on L84 and L93 to clarify the example tag intention.